src/main.rs

```diff
@@ -130,6 +132,34 @@ async fn start_servers(
     });
 }
 
+async fn schedule_background_jobs(
+    app_config: &snapchain::cfg::Config,
+    block_store: BlockStore,
+    local_state_store: LocalStateStore,
+) {
+    let sched = JobScheduler::new().await.unwrap();
+    let is_sync_complete = local_state_store.is_sync_complete().unwrap();
```

Comment:
I think this should go inside the job. Let's keep the scheduling logic as simple as possible.